res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -131,24 +168,20 @@ limitations under the License.
     }
 
     .mx_AutoHideScrollbar {
-        border-radius: 8px;
-    }
-
-    .mx_RoomView_messageListWrapper {
+        background: #fff;
         background-color: $background;
-        padding: 8px;
-        border-radius: inherit;
+        border-radius: 8px;
+        width: calc(100% - 16px);
```

Comment:
I assume `box-sizing` is no help to remove the `calc`...?

Reply:
It created all sort of funky regressions with the scrollbar. It's likely that a more elegant solution exists but I did not manage to figure it out yet

Reply:
as reference: https://github.com/matrix-org/matrix-react-sdk/pull/7105
(But gsouquet already found this anyways. Still wanted to link it here again.)